sdk/core/Azure.Core.Experimental/tests/DynamicData/DynamicJsonTests.cs

```diff
@@ -1046,6 +1047,53 @@ public void CanExplicitCastToDateTimeOffset()
             Assert.Throws<InvalidCastException>(() => { DateTimeOffset d = (DateTimeOffset)json.Foo; });
         }
 
+        internal static string FormatDateTimeOffset(DateTimeOffset d)
+        {
+            return d.ToUniversalTime().UtcDateTime.ToString("o");
+        }
+
+        [Test]
+        public void CanRoundTripUnixDateTime()
```

Comment:
@m-nash, I believe this represents the use case you were interested in -- e.g. "if you load something from the wire into DynamicData and then use that to roundtrip how do you know what format to write it back out as?"